src/ef-args.c

```diff
@@ -6,6 +6,12 @@
 #include <stdio.h>
 
 int argc_frame(int argc, const char *argv[], frame_t *f) {
+    
+    if(argc > FRAME_STACK_MAX) {
+        po("ERROR: Frame size is too big");
```

Comment:
Is this correct message? Is it the frame size or the stack size that is too big?

Reply:
it is actually stack size. The main idea is frame_t struct in ef.h can contain only 100 headers as of now. I'll change the comments. 
Thanks